index.html

```diff
@@ -405,8 +405,278 @@ <h4 class="title-small">Winning the talent war with a scalable tech skills platf
+                    </div>
+                </div>
+                <div>
+                    <p class="social-share-content">Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+                        Numquam
+                        animi dolore pariatur corrupti praesentium saepe, est sint dolores maiores ex.</p>
+                </div>
+                <div class="social-share-footer">
+                    <img class="footer-icon"
+                        src="https://www.pluralsight.com/etc/clientlibs/pluralsight/main/images/icons/twitter-reply-black.png"
+                        alt="">
+                    <img class="footer-icon"
+                        src="https://www.pluralsight.com/etc/clientlibs/pluralsight/main/images/icons/twitter-retweet-black.png"
+                        alt="">
+                    <img class="footer-icon"
+                        src="https://www.pluralsight.com/etc/clientlibs/pluralsight/main/images/icons/twitter-favorite-black.png"
+                        alt="">
+                </div>
+            </div>
+            <div>
+                <img src="/aby-website/images/chevron-right-social.svg" alt="">
+            </div>
+        </div>
+    </section>
+
+    <!--pluralsight-one-section-->
+    <section class="pluralsight-one-section">
+        <div class="pluralsight-one-container">
+            <div class="logo-size">
+                <img class="bottom-hero-logo" src="/aby-website/images/hero-bottom-logo.png" alt="">
+            </div>
+            <div>
+                <h1 class="hero-bottom-heading">See our growth, progress and evolution</h1>
+            </div>
+            <div>
+                <p class="hero-paragraph">At Pluralsight, we see first-hand every day how technology makes the
+                    impossible, possible. It’s why Pluralsight One exists: To advance our mission of democratizing
+                    technology skills, challenging assumptions about solutions and create significant, lasting social
+                    impact.
+                </p>
+            </div>
+            <div class="hero-button-container">
+                <div class="hero-button-size">
+                    <button class="view-impact-button">VIEW OUR 2020 IMPACT BOOK</button>
+                </div>
+                <div>
+                    <button class="for-nonprofit-button">FOR NONPROFITS</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- try for free section-->
+    <section class="try-for-free-section-container">
+        <div>
+            <h1 class="hero-bottom-heading">Insights that give you an edge</h1>
+        </div>
+        <div>
+            <button class="bottom-try-for-free-button">TRY FOR FREE</button>
+        </div>
+    </section>
+    <!-- footer section -->
+    <section class="footer-section">
+        <div class="footer-section-container">
+            <div class="footer-elements">
+                <div>
```

Comment:
empty div

Reply:
you meant to write a class name or remove the div? that div needed for lay out view